apps/pollbook/frontend/src/update_mailing_address_flow.tsx

```diff
@@ -48,7 +48,7 @@ function createBlankMailingAddress(): VoterMailingAddressChangeRequest {
     mailingAddressLine2: '',
     mailingAddressLine3: '',
     mailingCityTown: '',
-    mailingState: 'NH',
+    mailingState: '',
```

Comment:
for some reason the default mailing address was set to NH which, because NH didn't appear in the list, functioned exactly as if it were blank (as the function name suggests)